pkg/target/http.go

```diff
@@ -115,6 +193,8 @@ func (ht *HTTPTarget) Write(messages []*models.Message) (*models.TargetWriteResu
 	var errResult error
 
 	for _, msg := range safeMessages {
+		// TODO: THIS IS A PROBLEM!
```

Comment:
Just checking this is here deliberately?  I have no objections to merging it if you plan to address the problem at a later time.

Reply:
Actually, it's a mistake. Double-checking for others now.

Reply:
Ah ok - so it was a note I left to myself in a wip commit when I was adding the request timing. Simply forgot to remove/clean up this and a couple of other comments. Thanks for flagging, will clean up before merging. :)